.nuke/build.schema.json

```diff
@@ -330,6 +330,7 @@
               "CheckTestResultForProfilerWithSanitizer",
               "Clean",
               "CleanObjFiles",
+              "CleanTestLogs",
```

Comment:
Should these lines be added? It was autogenerated when i ran my instrumentation test